Pages/Shared/_Layout.cshtml

```diff
@@ -13,8 +12,19 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>@title</title>
-    <link rel="stylesheet" href="~/lib/bootstrap/dist/css/bootstrap.min.css" />
-    <link rel="stylesheet" href="~/css/site.css" />
+
+    <environment include="Development">
+        <link rel="stylesheet" href="~/lib/bootstrap/dist/css/bootstrap.css" />
+        <link rel="stylesheet" href="~/css/site.css" />
+    </environment>
+    <environment exclude="Development">
+        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+              asp-fallback-test-class="sr-only"
+              asp-fallback-test-property="position"
+              asp-fallback-test-value="absolute"
+              asp-fallback-href="~/lib/bootstrap/dist/css/bootstrap.min.css" />
```

Comment:
I see what this is going for and it's probably a good idea for a larger organization, but it also seems like it's going to be a bit of a testing headache. I don't want to deal with two completely separate upstream dependencies, and I'm willing to sacrifice a few ms of load time for that, especially given how good pipelining is. Can we change this back to the local lib only, not the cdn lib?